reframe/core/launchers/__init__.py

```diff
@@ -36,6 +36,14 @@ class JobLauncher(metaclass=_JobLauncherMeta):
     #: :default: ``[]``
     options = variable(typ.List[str], value=[])
 
+    #: Dictionary of environment variables to be passed via the job launcher
+    #: invocation. The keys are the variable names and the values are their
+    #: corresponding values.
+    #:
+    #: :type: :class:`Dict[str, str]`
+    #: :default: ``{}``
+    environment_variables = variable(typ.Dict[str, str], value={})
```

Comment:
To be consistent with the test's `env_vars`:

```suggestion
    env_vars = variable(typ.Dict[str, str], value={})
```